server/commands.lua

```diff
@@ -142,8 +142,15 @@ lib.addCommand('car', {
     restricted = "group.admin"
 }, function(source, args)
     if not args then return end
-    local netId = SpawnVehicle(source, args[Lang:t("command.car.params.model.name")], nil, true)
-    local plate = GetPlate(NetworkGetEntityFromNetworkId(netId))
+    local ped = GetPlayerPed(source)
+
+    local netId = qbx.spawnVehicle({
+        model = args[Lang:t("command.car.params.model.name")],
+        spawnSource = ped,
+        warp = ped,
+    })
```

Comment:
```suggestion
    local netId = qbx.spawnVehicle({
        model = args[Lang:t("command.car.params.model.name")],
        spawnSource = GetPlayerPed(source),
        warp = true,
    })
```
It's possible to do this instead